### PR TITLE
Add definition for XSD files

### PR DESCRIPTION
**Pull request by diazwatson**
## Scope of this PR

We set [labels](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#devdocs) based on the scope of changes with a PR.

- New Topic: Adds terms

Note the scope of this PR by leaving one of the options and deleting the others.

## Purpose of this PR (required)

Add definition for XSD noun

## Affected Terms (required)

- https://glossary.magento.com/xsd

### CODE REVIEW

File: src/terms/xsd.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  title: "XSD"
+  wordClasses:
+    - "noun"
+  tags:
+    - "programming"
+  synonyms:
+    - "XML Schema Definition"
+  relatedTerms: []
+---
+Abbreviation for "XML Schema Definition" specifies how to formally describe the elements in an Extensible Markup Language ([XML](https://glossary.magento.com/xml)) document.
```
Comment (jcalcaben) [inline src/terms/xsd.md]:
```suggestion
Abbreviation for "XML Schema Definition". This file describes the elements in an Extensible Markup Language ([XML](https://glossary.magento.com/xml)) document.
```

**Review by jcalcaben — CHANGES_REQUESTED** [review]
Just one minor suggestion

**Comment by diazwatson** [comment]
makes sense, thanks 👍

**Review by jcalcaben — APPROVED** [review]
Looks great! Thanks @diazwatson !

**Pull request merged**